### ISSUE TITLE
Please add keyboard navigation to grid

### ISSUE THREAD

**Issue by yacoob** [jigish/slate#389]
Can you please add a keyboard navigation for grid? Once grid is on screen, I should be able to use cursors to move a selection, select "top-left" position by pressing space, move to different section of the grid and select "bottom-right" position by pressing space again. That's how Moom does it - and I admit I miss it from Moom :D
